tests/test_args.py

```diff
@@ -14,62 +14,66 @@
 file_path = Path(__file__)
 test_folder = file_path.parent.absolute()
 
+def test_always_work():
+    true_var = True
+    print("This is not a real test")
+    assert true_var is True
 
-def test_arg_dry():
-    c = Config()
-    parser = c.init_argparse()
-    args = parser.parse_args(['--dry'])
-    assert args.dry is True
-    c.handle_args(args)
-    assert c.dry is True
+# def test_arg_dry():
```

Comment:
umm???

Reply:
This was a working PR to share code with Ola. The merge is into his branch so he can see my edits. The intent is to never merge this. I should have marked WIP.